hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/token/HapiPartitionCreate.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (C) 2020-2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.services.bdd.spec.transactions.token;
+
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.SUCCESS;
+
+import com.google.common.base.MoreObjects;
+import com.hedera.services.bdd.spec.HapiSpec;
+import com.hedera.services.bdd.spec.transactions.HapiTxnOp;
+import com.hedera.services.bdd.spec.transactions.TxnUtils;
+import com.hedera.services.bdd.suites.HapiSuite;
+import com.hederahashgraph.api.proto.java.HederaFunctionality;
+import com.hederahashgraph.api.proto.java.Key;
+import com.hederahashgraph.api.proto.java.TokenCreatePartitionDefinitionTransactionBody;
+import com.hederahashgraph.api.proto.java.TokenID;
+import com.hederahashgraph.api.proto.java.Transaction;
+import com.hederahashgraph.api.proto.java.TransactionBody;
+import com.hederahashgraph.api.proto.java.TransactionResponse;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Consumer;
+import java.util.function.Function;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class HapiPartitionCreate extends HapiTxnOp<HapiPartitionCreate> {
+
+    static final Logger log = LogManager.getLogger(HapiPartitionCreate.class);
+
+    private String token;
+
+    public HapiPartitionCreate(final String token) {
+        this.token = token;
+    }
+
+    protected HapiPartitionCreate self() {
+        return this;
+    }
+
+    @Override
+    public HederaFunctionality type() {
+        return HederaFunctionality.TokenCreatePartition;
+    }
+
+    private Optional<String> parent_token_id = Optional.empty();
+    private Optional<String> memo = Optional.empty();
```

Comment:
It might be better to re-use the [memo](https://github.com/hashgraph/hedera-services/blob/735e56a00ce1ceabe412f026732a2fb4b98751d4/hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/HapiSpecOperation.java#L130) which is already defined from the parent HapiSpecOperation class here ?